src/main/java/de/rabea/Game/Board.java

```diff
@@ -23,11 +23,25 @@ public Board(Cell[] cells) {
         return cells;
     }
 
-    public Cell[] placeMark(int position, Cell mark) {
-        cells[position] = mark;
+    public Cell[] placeMark(int position, Mark mark) {
+        cells[position] = mark.convertToCell();
         return cells;
```

Comment:
Why return the internal representation? I expect a method named placeMark to be void and to just have a side-effect.